packages/eth2.0-params/src/interface.ts

```diff
@@ -3,7 +3,7 @@
  */
 import BN from "bn.js";
 
-import {Epoch, Gwei, Slot, Shard, number64} from "../types";
+import {Epoch, Gwei, Slot, Shard} from "@chainsafe/eth2.0-types";
```

Comment:
I cannot put comment anywhere else, but param presets should also be exported in index to avoid weird imports